src/util/segmentation/brush/getNewColorLUT.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * GetNewColorLUT - Generates an array of RGB colors of length numberOfColors.
+ *
+ * @param  {Number} numberOfColors = 255 The number of colors to generate
+ * @returns {Number[][]}           The array of RGB values.
+ */
+export default function getNewColorLUT(numberOfColors = 255) {
```

Comment:
This may sound silly, but I would like it if we included some more info around what a color LUT is and when/why you might use/need one. It took me the longest time when I first started to get how/why/when

Reply:
Yeah that's totally fair :)